crates/continuations/src/verifier/common/non_leaf.rs

```diff
@@ -50,37 +50,38 @@ impl<C: Config> NonLeafVerifierVariables<C> {
             let proof_vm_pvs = self.verify_internal_or_leaf_verifier_proof(builder, &proof);
 
             assert_single_segment_vm_exit_successfully(builder, &proof);
-            builder.if_eq(i, RVar::zero()).then_or_else(
-                |builder| {
-                    builder.assign(&pvs.app_commit, proof_vm_pvs.vm_verifier_pvs.app_commit);
-                    builder.assign(
-                        &leaf_verifier_commit,
-                        proof_vm_pvs.extra_pvs.leaf_verifier_commit,
-                    );
-                },
-                |builder| {
-                    builder.assert_eq::<[_; DIGEST_SIZE]>(
-                        pvs.app_commit,
-                        proof_vm_pvs.vm_verifier_pvs.app_commit,
-                    );
-                    builder.assert_eq::<[_; DIGEST_SIZE]>(
-                        leaf_verifier_commit,
-                        proof_vm_pvs.extra_pvs.leaf_verifier_commit,
-                    );
-                },
-            );
+
+            // builder.if_eq(i, RVar::zero()).then_or_else(
```

Comment:
nitpicks: some explanation is better (e.g. future maybe need abandon curve based continuation and probably switch back to merkle tree based solution)

Reply:
We will bring this assertion back. Only the merkle tree root consistency will be removed.